src/components/atoms/Select/Select.tsx

```diff
@@ -1,3 +1,8 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable @typescript-eslint/no-use-before-define */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable no-restricted-syntax */
+
```

Comment:
Oupsi didn't pay attention to this yesterday but why ignoring all theses linter rules? 🤔

Reply:
because i could not find any satisfaction for the linter

Reply:
but i try too solve this with the refactoring